ftdetect/sml.vim

```diff
@@ -1,2 +1,5 @@
 " SML signature files
-au BufRead,BufNewFile *.sig setlocal filetype=sml
+augroup VimBetterSMLFtDetect
+  autocmd!
+  autocmd BufRead,BufNewFile *.sig setlocal filetype=sml
+augroup END
```

Comment:
Oh, huh. I suppose that's a valid point. I could believe that it depended on the implementation of the plugin manager.